### PR TITLE
#29 in upload images

### PR DESCRIPTION
**Pull request by dhanzy**
### What this PR does (required):
- Profile Page Modification
- Enables user to change profile picture

### Screenshots / Videos (required):
- Screenshot (31)

### Any information needed to test this feature (required):
- Login / Sign up and click on the edit profile on the dashboard or navbar

### Any issues with the current functionality (optional):
- None

### CODE REVIEW

File: client/src/components/SidePanel/SidePanel.tsx [hunk 1]
```diff
@@ -37,13 +38,26 @@ function SidePanel({ children }: PanelProps): JSX.Element {
             <Paper elevation={3} className={classes.paperStyle}>
               <Box py={5}>
                 <List component="nav" aria-label="main mailbox folders" className={classes.navSide}>
-                  <ListItem button component={Link} to="/profile">
+                  <ListItem
+                    button
+                    component={Link}
+                    to="/profile"
+                    selected={(location.pathname == '/profile' && true) || (location.pathname == '/profile/' && true)}
```
Comment (rajivtitus) [inline client/src/components/SidePanel/SidePanel.tsx]: We will always be routing to '/profile', so do we need both operands here? :)

File: client/src/components/SidePanel/SidePanel.tsx [hunk 2]
```diff
@@ -37,13 +38,26 @@ function SidePanel({ children }: PanelProps): JSX.Element {
...
                     <ListItemText primary="Profile"></ListItemText>
                   </ListItem>
-                  <ListItem button component={Link} to="/personal-info">
+                  <ListItem
+                    button
+                    component={Link}
+                    to="/personal-info"
+                    selected={
+                      (location.pathname == '/personal-info' && true) ||
+                      (location.pathname == '/personal-info/' && true)
```
Comment (rajivtitus) [inline client/src/components/SidePanel/SidePanel.tsx]: Same as above!

**Review by rajivtitus — APPROVED** [review]
LGTM! 

Just a couple of small comments, please have a look.

**Comment by dhanzy** [comment]
Okay @rajivtitus . Will do that

**Pull request merged**